style-sheets/primevue-components-overrides.scss

```diff
@@ -391,7 +392,57 @@ label {
 .#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-footer {
     padding: 0;
 }
+
 /* ------------------------------------ primevue table --------------------------------- */
+
+.custom-access-table {
+    background: transparent;
+    border-radius: 0.25rem 0.25rem 0 0;
+    border: 0.125rem solid map.get(lightTheme.$light-theme, 'border-subtle-00');
+}
+
+.custom-table-header {
+    padding: 1rem 1rem 1.5rem;
+    h3 {
+        @extend %heading-03;
+        margin: 0;
+        padding: 0;
+    }
+
+    span {
+        @extend %body-compact-01;
+        margin: 0;
+        padding: 0;
+        color: map.get(lightTheme.$light-theme, 'text-secondary');
+    }
+}
+
+.span-icon {
```

Comment:
also this one I'm not sure where we use it, but we only put primevue component style change in this file.

Reply:
This is for giving some space between the user name and the avatar icon.